lab-trent/lib/resourcemanager.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
Fairly simple execution here. I'd suggest implementing a promise-based return for each of these, as well as handling some basic errors. I might not have looked over them yet if they're elsewhere in the PR, but I think it's appropriate to have a layer of error handling here as well.